packages/mui-base/src/SliderUnstyled/SliderUnstyled.tsx

```diff
@@ -76,7 +96,7 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
 
   // all props with defaults
   // consider extracting to hook an reusing the lint rule for the variants
-  const ownerState = {
+  const partialOwnerState: PartialSliderUnstyledOwnerState = {
```

Comment:
```suggestion
  const partialOwnerState: Partial<SliderUnstyledOwnerState> = {
```
Can we do this instead of creating a new type?

Reply:
Or better name the interface to be specific to the props it contains.